src/store/features/blocks/blocks.ts

```diff
@@ -0,0 +1,97 @@
+import tbConf, {ToolboxConfiguration} from "../../../toolboxconfig"
+import {produce} from 'immer'
+import {MutableRefObject} from "react";
+
+// Define actions
+const TOGGLE_CATEGORY = 'category/toggle'
```

Comment:
To ease mergeability with the Codecast existing project, could you use please an enum here? For example:

    export enum ActionTypes {
        BlocksCategoryToggle = 'Blocks.Category.Toggle',
        BlocksBlockDescriptionToggle = 'Blocks.Block.Description.Toggle'
    }

That will be much easier for integration afterwards. thanks!

Reply:
Understood!